frontend/cypress/integration/common/apps.ts

```diff
@@ -85,10 +85,14 @@ And('user sees Details information for Apps', () => {
   });
 });
 
-Then('user only sees {int} apps', (sees: number) => {
+Then('user only sees the filtered apps', () => {
+  let count:number;
+  cy.request('GET', "/api/namespaces/bookinfo/apps").should((response) =>{
```

Comment:
The method should not contain hardcoded values such as "bookinfo" and "reviews", they need to be as param.
What if you just change the conflicting k8s tests to use "ratings" instead of "reviews"? They should not collide.